vcpkg/overlay/qca/portfile.cmake

```diff
@@ -50,6 +51,8 @@ if("botan" IN_LIST FEATURES)
 else()
     list(APPEND QCA_OPTIONS -DWITH_botan_PLUGIN=no)
 endif()
+    list(APPEND QCA_OPTIONS -DWITH_gnupg_PLUGIN=no)
+    list(APPEND QCA_OPTIONS -DWITH_ossl_PLUGIN=no)
```

Comment:
@nirvn are you aware if this is essential for 2fa?

Reply:
@m-kuhn , I do not know the answer :(

Reply:
There's no problem in keeping the plugins activated, so the question is irrelevant